### PR TITLE
chore(dependencies): update dsp-api to 13.4.0

### PR DESCRIPTION
**Pull request by mdelez**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,35 +1 @@
...
-  ],
-  "files": [
-    "index.ts"
-  ],
-  "include": [
-    "src/**/*.ts"
-  ]
-}
\ No newline at end of file
+{"compilerOptions":{"declaration":true,"emitDecoratorMetadata":true,"esModuleInterop":true,"experimentalDecorators":true,"lib":["es6","dom"],"module":"commonjs","removeComments":false,"target":"es5","resolveJsonModule":true,"sourceMap":true,"strictNullChecks":true,"noImplicitAny":true,"outDir":"build"},"exclude":["build","coverage","dist","node_modules","**/*.spec.ts","test-framework"],"files":["index.ts"],"include":["src/**/*.ts"]}
```
Comment (kilchenmann) [inline tsconfig.json:1]: Something went wrong here. Does it come from `npm run prepare-dev-publication`? You should revert the changes...

Comment (mdelez) [reply tsconfig.json]: yeah, seems like it. reverted in 5df2d0d

**Review by kilchenmann — APPROVED** [review]
I updated the format of tsconfig.json. Now it's fine 😉
